puns.json

```diff
@@ -190,5 +190,13 @@
   {
     "pun": "Q: Why were the Javascript plumbers delayed?",
     "punchline": "A: Because they had to await async."
+  },
+  {
+    "pun": "Q: Why do Java developer wear glasses?",
+    "punchline": "A: Because they don't C#!"
+  },
```

Comment:
This one is already in the list, please remove it.
https://github.com/AlexLakatos/computer-puns/blob/a75fef11c16b034fe812acc8d8157365ddd0927a/puns.json#L71